rooms.js

```diff
@@ -263,7 +263,19 @@ class Room {
 	}
 
 	get metadata() {
-		let settings = {title: this.title, id: this.id, privacy: (this.isPrivate === true ? 'secret' : this.isPrivate === 'hidden' ? 'hidden' : 'public'), type: this.type, modchat: this.modchat, modjoin: this.modjoin, auth: (this.auth || {}), uno: (!this.unoDisabled), tournaments: (this.toursEnabled === '%' ? '%' : this.toursEnabled ? '@' : '#'), hangman: (!this.hangmanDisabled)};
+		let settings = {
+			// settings
+			privacy: (this.isPrivate === true ? 'secret' : this.isPrivate === 'hidden' ? 'hidden' : 'public'),
+			modchat: this.modchat,
+			modjoin: this.modjoin,
+			auth: (this.auth || {}),
```

Comment:
This is way too big of a thing to send automatically to clients who didn't ask for it.